### PR TITLE
Update Search docs

### PR DESCRIPTION
**Pull request by marcusolsson**
This PR updates the docs for the Search plugin.

### CODE REVIEW

File: en/Plugins/Search.md [hunk 1]
```diff
@@ -1,82 +1,98 @@
...
+| `file:`         | Find text in filename.<p/>Example: `file:.jpg` or `file:202209`.                                                                                                                                                     |
+| `path:`         | Find text in file path.<p/>Example: `path:"Daily notes/2022-07"`.                                                                                                                                                    |
+| `content:`      | Find text in file content.<p/>Example: `content:"happy cat"`.                                                                                                                                                        |
+| `match-case:`   | Case-sensitive match.<p/>Example: `match-case:HappyCat`.                                                                                                                                                             |
+| `ignore-case:`  | Case-insensitive match.<p/>Example: `ignore:case:ikea`.                                                                                                                                                              |
```
Comment (ericaxu) [inline en/Plugins/Search.md:46]: "ignore:case" => "ignore-case" in example

File: en/Plugins/Search.md [hunk 2]
```diff
@@ -1,82 +1,98 @@
...
 
-```query
-embed OR search
-```
+You can even combine regular expressions with search operators:
+
+- `path:/\d{4}-\d{2}-\d{2}/` returns files with a date in the file path.
 
-## Search syntax
+For more information on how to write regular expressions, refer to [Regular expressions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions)
```
Comment (ericaxu) [inline en/Plugins/Search.md:65]: Love the link to Mozilla docs! It's unlikely that we can teach people how to use RegExp within our docs haha

File: en/Plugins/Search.md [hunk 3]
```diff
@@ -1,82 +1,98 @@
...
 
-## Search history
+- `meeting work` returns files that contain both `meeting` and `work`.
+- `meeting OR work` returns files that contain either `meeting` or `work`.
 
-Obsidian will remember your most recently used search queries.
+You can even combine the two in the same search term.
 
-These queries will be presented to you when your search query is empty. You can click on any one of them to search for it again, allowing you to easily re-run previous queries.
+- `meeting work OR meetup personal` returns files for work meetings and personal meetups.
```
Comment (ericaxu) [inline en/Plugins/Search.md:21]: Did you reverse the order of keywords to signify the order of keywords within a document doesn't matter?

Comment (marcusolsson) [reply en/Plugins/Search.md:21]: That was the idea at least :) Maybe we can find a better example if it's too contrived.

File: en/Plugins/Search.md (hunk 1 above)
Comment (marcusolsson) [reply en/Plugins/Search.md]: Good eye! 👁️

**Comment by ericaxu** [comment]
LGTM! 🚢

**Pull request merged**